packages/api/lib/parse-pubsub-payload.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+const assert = require('assert');
+const firebaseEncode = require('./firebase-encode');
+
+function buildInfo (metadata) {
+  const [org, repo] = metadata.repo.split('/');
+  const match = metadata.buildURL.match(/\[Sponge\]\((?<url>.*)\)/);
+  return {
+    repoId: firebaseEncode(decodeURIComponent(metadata.repo)),
+    organization: org,
+    timestamp: new Date(),
+    url: match ? match.groups.url : `https://github.com/${metadata.repo}`,
+    environment: {
+      matrix: '{"kokoro":""}',
+      os: 'Linux',
```

Comment:
Should we make a note somewhere that this is essentially fake data? Or instead of hard-coding 'Linux' we could say `os: 'undefined'` or something like that?

Reply:
I think a comment would be good, I believe our servers are all `Linux`, but that's obviously not the case  for someone else using this.